src/apps/testapps/testGridDisk.c

```diff
@@ -329,6 +329,19 @@ SUITE(gridDisk) {
         t_assert(out == origin, "Moving to self goes to self");
     }
 
+    TEST(h3NeighborRotations_invalid) {
+        // This is undefined behavior, but it's helpful for it to make sense.
+        H3Index origin = 0x811d7ffffffffffL;
```

Comment:
Anything special about this particular H3 index?

Reply:
No - I just needed a known good index as I wanted the only rotations to be under test.

Reply:
For my info, is there any difference between indexes with the `L` suffix or without? Should we be consistent?

Reply:
All valid H3 indexes would only be valid integers as 64 bit, so no I don't believe so.